src/docs/src/config/scanner.rst

```diff
@@ -0,0 +1,231 @@
+.. WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+.. License for the specific language governing permissions and limitations under
+.. the License.
+
+.. default-domain:: config
+.. highlight:: ini
+
+=======
+Scanner
+=======
+
+Configure background scanning plugins.
+
+.. _config/scanner:
+
+.. versionadded:: 3.4
+
+Scanner Options
+===============
+
+.. config:section:: couch_scanner :: Scanner Options
+
+    .. config:option:: interval_sec
+
+        How often to check for configuration changes and start/stop plugins.
+        The default is 5 seconds. ::
+
+            [couch_scanner]
+            interval_sec = 5
+
+    .. config:option:: min_penalty_sec
+
+        Minimum time to force a plugin to wait before running again after a
+        crash. Defaults to 30 seconds. ::
+
+            [couch_scanner]
+            min_penalty_sec = 30
+
+    .. config:option:: max_penalty_sec
+
+        Maximum time to force a plugin to wait after repeated crashes. The
+        default is 8 hours (in seconds). ::
+
+            [couch_scanner]
+            min_penalty_sec = 28800
+
+    .. config:option:: heal_threshold_sec
+
+        If plugin runs successfully without crashing for this long, reset its
+        repeated error count. Defaults to 5 minutes (in seconds). ::
+
+            [couch_scanner]
+            heal_threshold_sec = 300
+
+    .. config:option:: db_rate_limit
+
+        Database processing rate limit. This will also be the rate at which
+        design documents are fetched. The rate is shared across all running
+        plugins. ::
+
+            [couch_scanner]
+            db_rate_limit = 50
+
+    .. config:option:: shard_rate_limit
+
+       Limits the rate at which plugins may open db shard files on a node. The
+       rate is shared across all running plugins. ::
+
+            [couch_scanner]
+            db_shard_limit = 50
+
+    .. config:option:: doc_rate_limit
+
+        Limit the rate at which plugins open documents. The rate is shared
+        across all running plugins. ::
+
+            [couch_scanner]
+            doc_rate_limit = 500
+
+.. config:section:: couch_scanner_plugins :: Enable Scanner Plugins
+
+    .. config:option:: $plugin
+
+        Which plugins are enabled. By default plugins are disabled. ::
+
+            [couch_scanner_plugins]
+            couch_scanner_plugin_ddoc_features = false
+            couch_scanner_plugin_find = false
+
+.. config:section:: $plugin :: General Plugin Settings
+
+These settings apply to all the plugins. Some plugins may also have individual
+settings in their ``[$plugin]`` section.
+
+    .. config:option:: after
+
+        Run plugin on or after this time. The default is to run once after the
+        node starts. Times are in UTC. Possible time formats are: unix seconds
+        (ex. ``1712338014``) or date/time: ``YYYY-MM-DD``, ``YYYY-MM-DDTHH``,
+        ``YYYY-MM-DDTHH:MM``. Times are in UTC. ::
```

Comment:
`Times are in UTC.`
It appears twice, maybe we can remove one of them?

Reply:
Good find, thanks! I'll remove one of them

Reply:
Removed the first one and left the last one in.